templates/etc/sysconfig/archivematica-mcp-client.j2

```diff
@@ -6,3 +6,7 @@ PYTHONPATH={{ archivematica_src_am_mcpclient_app }}:{{ archivematica_src_am_comm
 {% for key, value in archivematica_src_am_mcpclient_environment.iteritems() %}
 {{ key }}={{ value }}
 {% endfor %}
+
+{% if clamav_uses_file_socket is not defined %}
+ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_CLAMAV_SERVER=localhost:3310
```

Comment:
(note to myself) 
- From the [MCP Client config docs](https://github.com/artefactual/archivematica/tree/stable/1.7.x/src/MCPClient/install) : 
ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_CLAMAV_SERVER:
Description: configures the clamdscanner backend so it knows how to reach the clamd server via UNIX socket (if the value starts with /) or TCP socket (form host:port, e.g.: myclamad:3310).
Default: /var/run/clamav/clamd.ctl

- Centos/RH clamav server default config is using TCP sockets on port 3310
- ubuntu clamav server default config is using Unix socket /var/run/clamav/clamd.ctl